src/lib/viewers/controls/annotations/AnnotationsButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ButtonHTMLAttributes } from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import { AnnotationMode } from './types';
+import './AnnotationsButton.scss';
+
+export type Props = Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onClick'> & {
+    children?: React.ReactNode;
+    className?: string;
+    isActive?: boolean;
+    isEnabled?: boolean;
+    fileId: string;
+    onClick?: (mode: AnnotationMode) => void;
+    mode: AnnotationMode;
+};
+
+export default function AnnotationsButton({
+    children,
+    className,
+    isEnabled = true,
+    isActive = false,
+    fileId,
+    onClick = noop,
+    mode,
+    ...rest
+}: Props): JSX.Element | null {
+    if (!isEnabled) {
+        return null;
+    }
+
+    return (
+        <button
+            className={classNames('bp-AnnotationsButton', className, {
+                'bp-is-active': isActive,
+            })}
+            data-resin-fileid={fileId}
```

Comment:
Could this resin tag be applied at a higher level, like `ControlsBar` to avoid having to propagate it down to each button?

Reply:
Not sure. This is how the controls are built today, so I'm just emulating that.

Reply:
It works on higher levels. I'll move it up. Thanks for the feedback!